modules/spo-applying-profiles.adoc

```diff
@@ -0,0 +1,180 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-seccomp.adoc
+// * security/security_profiles_operator/spo-selinux.adoc
+
+ifeval::["{context}" == "spo-seccomp"]
+:seccomp:
+:type: seccomp
+:kind: SeccompProfile
+endif::[]
+ifeval::["{context}" == "spo-selinux"]
+:selinux:
+:type: SELinux
+:kind: SelinuxProfile
+endif::[]
+
+:_content-type: PROCEDURE
+[id="spo-applying-profiles_{context}"]
+= Applying {type} profiles to a pod
+
+Create a Pod to apply one of the created profiles.
+
+ifdef::selinux[]
+For {type} profiles, the namespace must be labelled to allow link:https://kubernetes.io/docs/concepts/security/pod-security-standards/[privileged] workloads.
+
+endif::[]
+.Procedure
+
+ifdef::seccomp[]
+. Create a Pod object that defines a `securityContext`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-pod
+spec:
+  securityContext:
+    seccompProfile:
+      type: Localhost
+      localhostProfile: operator/my-namespace/profile1.json
+  containers:
+    - name: test-container
+      image: nginx
```

Comment:
can we use another image instead? One image that won't report "CrashLoopBackOff"? Thanks.
```
$ oc get pod
NAME       READY   STATUS             RESTARTS      AGE
test-pod   0/1     CrashLoopBackOff   2 (29s ago)   46s
[xiyuan@MiWiFi-RA69-srv 412]$ oc logs pod/test-pod
/docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
/docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
/docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
10-listen-on-ipv6-by-default.sh: info: can not modify /etc/nginx/conf.d/default.conf (read-only file system?)
/docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
/docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
/docker-entrypoint.sh: Configuration complete; ready for start up
2022/12/08 07:55:31 [warn] 1#1: the "user" directive makes sense only if the master process runs with super-user privileges, ignored in /etc/nginx/nginx.conf:2
nginx: [warn] the "user" directive makes sense only if the master process runs with super-user privileges, ignored in /etc/nginx/nginx.conf:2
2022/12/08 07:55:31 [emerg] 1#1: mkdir() "/var/cache/nginx/client_temp" failed (13: Permission denied)
nginx: [emerg] mkdir() "/var/cache/nginx/client_temp" failed (13: Permission denied)
```

Reply:
I think for OCP `quay.io/security-profiles-operator/test-nginx-unprivileged:1.21` would be a better example.

(This is again my fault, I was on a crusade to eliminate the use of priviliged containers from SPO upstream tests a couple of months ago because then the tests wouldn't run on OCP, but I never updated the upstream docs)